src/app/public/modules/date-pipe/fuzzy-date.pipe.ts

```diff
@@ -0,0 +1,91 @@
+import {
+  OnDestroy,
+  Pipe,
+  PipeTransform
+} from '@angular/core';
+
+import {
+  SkyAppLocaleProvider
+} from '@skyux/i18n';
+
+import {
+  Subject
+} from 'rxjs/Subject';
+
+import 'rxjs/add/operator/takeUntil';
+
+import {
+  SkyFuzzyDate
+} from '../datepicker/fuzzy-date';
+
+import {
+  SkyFuzzyDateService
+} from '../datepicker/fuzzy-date.service';
+
+/**
+ * Formats a fuzzy date value according to locale rules.
+ * You can construct a format string using symbols to specify the components
+ * of a date-time value, as described in the following table.
+ * These fields are based on the moment.js values found here: https://momentjs.com/docs/#/displaying/
```

Comment:
`These fields are based on the [moment.js values](https://momentjs.com/docs/#/displaying/).`

Reply:
done.